server/util-server.js

```diff
@@ -1132,13 +1132,17 @@ if (process.env.TEST_BACKEND) {
  */
 module.exports.axiosAbortSignal = (timeoutMs) => {
     try {
+        // Just in case, as 0 timeout here will cause the request to be aborted immediately
+        if (!timeoutMs || timeoutMs <= 0) {
+            timeoutMs = 5000;
+        }
```

Comment:
Due to https://github.com/louislam/uptime-kuma/issues/4027, I want to revert `axiosAbortSignal` change, as I really can't find the root cause.

But I honestly don't understand, how do they bypass this "Just in case" limit.

Reply:
Can we collect a bit more information and see if we can find the cause first? I don't think it's very wide spread, and we still don't know what configuration is people having this problem.

Reply:
We can also patch the monitors' config to see if that helps.